configuration/drugs/drugs.csv

```diff
@@ -1,323 +1,321 @@
+b61b29d2-41ee-4079-8fbb-a054596b7ae1,,Acétaminophene Gtts 60mg/ml,3440c780-b5ff-4b47-82e2-e0b8fc16bc89,94ecbbb3-71b7-407b-bac4-3279abbf1b69,60mg/ml,
+b5b194a3-efc9-47b0-8406-db87092c23e2,,Rapidal Plus 162.5 mg  + 100 mg,46a9fa37-d6f2-4f72-8a16-c1a76008f508,94ecbbb3-71b7-407b-bac4-3279abbf1b69,162.5 mg  + 100 mg,
+4eb11496-3275-4087-ab5c-b1e4ea885e99,,Acide Folique Co 5mg,9486be67-f05f-44a3-8962-0746f0904f1b,fdbe907b-6abc-41dc-9599-2d84105e4063,5mg,
+123c4342-552a-4db9-bb0b-a7d0bd2e5849,,Acyclovir Co 400mg,783aff6f-c427-4511-8df4-872b87fad050,fdbe907b-6abc-41dc-9599-2d84105e4063,400mg,
+45aaf344-e567-4716-be52-4a7a642305c1,,Acyclovir Sirop 200mg,783aff6f-c427-4511-8df4-872b87fad050,94ecbbb3-71b7-407b-bac4-3279abbf1b69,200mg,
+a74fe93c-b023-4159-8bd6-5d40a0909636,,Acyclovir creme 3%,783aff6f-c427-4511-8df4-872b87fad050,0689bb6e-9371-4a0f-ad1c-63c135b1714d,3%,
+22269776-0866-4d7f-b4f6-901d31531f83,,Acyclovir ophtalmique 5%,783aff6f-c427-4511-8df4-872b87fad050,0689bb6e-9371-4a0f-ad1c-63c135b1714d,5%,
+77addc86-5036-4a82-b722-826635e1ce0b,,Adrenaline Injection 1 mg,d770fc0a-1482-4078-b0cd-9a91d9afe0e5,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,1 mg,
+37302e59-9824-4ec3-a8f7-6b491a5ee183,,Albendazole Co 400mg,8f668b49-584f-4cba-b0f2-817a8aaf50c4,fdbe907b-6abc-41dc-9599-2d84105e4063,400mg,
+7bf6328a-4194-484e-846f-a98bd41634e8,,Albendazole Sirop 400 mg/10 ml,8f668b49-584f-4cba-b0f2-817a8aaf50c4,94ecbbb3-71b7-407b-bac4-3279abbf1b69,400 mg/10 ml,
+39a96958-781f-4d84-a9e8-56f6dfbf9061,,Albuterol Inhalation 2.5mg/3ml,445f380a-f693-4100-9c9a-e43b73635039,658c1b9c-51c8-4734-8bf2-80e750c7889f,2.5mg/3ml,
+c3fb68d0-ba3d-4b94-bcb8-b2286d9b0358,,Alpha Métyl Dopa Co 250mg,84bb5c70-878b-45c3-96e7-3282e4121e02,fdbe907b-6abc-41dc-9599-2d84105e4063,250mg,
+58a9f6d0-1092-4c9c-b195-fbd7e43b39dd,,Aminovit Gtts,e0cf9a43-ec8a-4b41-a856-688983c1a974,94ecbbb3-71b7-407b-bac4-3279abbf1b69,N/A,
+c91f7481-cb7b-40ad-92cc-e8f54534de29,,Amlodipine Co 10mg,93491208-5dc1-47e5-9f3c-7bf99919f13a,fdbe907b-6abc-41dc-9599-2d84105e4063,10mg,
+1eebc619-5593-44be-9057-041df86b06ae,,Amoxycilline Sirop 125 mg,627a6041-ad81-4968-8adf-2f49aec4433b,94ecbbb3-71b7-407b-bac4-3279abbf1b69,125mg,
+d879219a-a74d-4a4a-a893-ebeb5594c629,,Amoxycilline Co 500mg,627a6041-ad81-4968-8adf-2f49aec4433b,fdbe907b-6abc-41dc-9599-2d84105e4063,500mg,
+a0f54b5b-0eda-49aa-82be-3634df0de78f,,Ampiciline Injection vial 1g,f9f69437-669d-4a3e-930b-b36994a2f9d4,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,vial 1g,
+c3e48c79-8b02-4442-9480-8f656a353b89,,Othorinol Gouttes 30mg,07470e7f-f959-4344-89f6-8e6a0f442bef,90a3f930-2eef-4a2e-8427-5359607c7b13,30mg,
+8e89914e-560b-4ac1-851c-ffeb54cb677f,,Apetamine Sirop,66de98f3-03e1-4733-9534-1f59dee19429,94ecbbb3-71b7-407b-bac4-3279abbf1b69,N/A,
+a42089dd-e91e-49a3-8c17-8571f2e5a404,,Apressoline Injection vial 20 mg,a6ba4400-1db2-4d47-854f-3342204995ba,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,vial 20 mg,

```

Comment:
Let's make this 'Apressoline Injection vial 20mg' instead of 'Apressoline Injection vial 20 mg'